HelpSource/Classes/Splay.schelp

```diff
@@ -19,7 +19,11 @@ An amplitude multiplier for all channels
 argument:: center
 Shift the centre of the distribution.
 argument:: levelComp
-
+level compensation based on number of inArray channels:
+false or floating point 0.0 is no compensation,
+true or floating point 0.5 is equal power compensation,
+and floating point 1.0 is equal amplitude compensation.
```

Comment:
see my suggestion in the LevelComp help file about using `list::` formatting to render this nicely.